### DISCUSSION TITLE
BLIP tiled sampler... I need it.

### DISCUSSION THREAD

**Discussion by Ferniclestix** [WASasquatch/was-node-suite-comfyui#160] [Ideas]
So, messing around with blip image analysis to create prompts and.... weell i kinda built this madcap thing. 
![dunno](https://github.com/WASasquatch/was-node-suite-comfyui/assets/128430314/466fa2c5-5536-449f-a8b4-40529b9574fb)

![Screenshot 2023-08-13 225644](https://github.com/WASasquatch/was-node-suite-comfyui/assets/128430314/bbd7b40f-9412-411e-9498-2702ed5311c2)

workflow
https://drive.google.com/file/d/1NY8AlgP1_gAOAdnrS4sKOzWPbJLl5kUk/view?usp=sharing

basically splitting an image into 9 performing blip on each segment and plugging it back in as conditioning and sampling each using the blip prompt.,

The idea is to basically refine a tiled sampler so that less hallucinations go down in each segment.  only problem is, merging them back together is difficult because the 'stitch' node does not emulate tiled samplers and doesnt heal seams. so I was like, man if this was in one sampler... it might be cool.
Because each tiled image has its own specific prompt its less likely to try and reproduce teh rest of the image in each tile at higher denoise.

And so im suggesting it seeing as most of the system is built using WAS nodes :P  

Kind of an 'interrogative tiled sampler' or blip tiled sampler.

I mean... seems to work not too terribly as is other than the seams.

**Comment by WASasquatch** [comment]
This is similar to the "detail" prompt I had in easy diffusion colab, and actually work better for you. Basically instead of trying to prompt subject, just prompt detail modifiers to improve the fielding of the scene

`sharp focus, highly detailed` and heavy negatives on stuff like blur, pixelation, etc.

**Reply by Ferniclestix** [reply]
Yeah, I mean, thats kind of the goal. Been batching a bunch of images using it to see where it might fall down. would need something like. some tuning that stops it going too far outside the original prompt as it does hallucinate a little if you don't merge with the original conditioning.
Was thinking of using text concat and forcing the weight of the added main prompt at a much lower influence to reduce influence of any hallucinatory prompts caused by blip. (getting surfers when it looks really close to rock walls XD)
of course I could tune it at the BLIP source too by filtering out words and maybe tuning it using some detail words.
